azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/WebAppBaseImpl.java

```diff
@@ -74,7 +74,7 @@
             WebAppBase.Update<FluentT>,
             WebAppBase.UpdateStages.WithWebContainer<FluentT> {
 
-    SiteConfigResourceInner siteConfig;
+    protected SiteConfigResourceInner siteConfig;
```

Comment:
ah, confused with C# - thought that protected was less visible than package-internal.
fixing.